src/components/EmptyState/EmptyState.css

```diff
@@ -1,34 +1,347 @@
 .empty_state_wrapper {
-    height: 100vh;
-    width: 100%; 
-    display: flex;
-    align-items: center;
-    z-index: -1;
-    position: absolute;
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  z-index: -1;
+  position: absolute;
 }
 
 .empty_state {
-    display: flex;
-    flex-direction: column;
-    text-align: center;
-    align-items: center;
-    margin: 0 auto;
-  }
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  margin: 0 auto;
+}
+
+.empty_state h1 {
+  font-family: 'gilroyextrabold', Helvetica, Sans-Serif;
+  font-weight: normal;
+  margin: 10px auto 20px auto;
+  font-size: 27px;
+  letter-spacing: 1.05px;
+}
+
+.empty_state p {
+  font-family: 'rennerbook', Helvetica, Sans-Serif;
+  font-weight: normal;
+  margin: 6px auto;
+  color: #4d4d4d;
+}
+
+.empty_state img {
+  height: 220px;
+  width: 220px;
+  text-align: center;
+}
+
+.clearfix {
+  display: inline-block;
+}
+
+.clearfix::after {
+  content: '';
+  visibility: hidden;
+  display: inline-block;
+  font-size: 0;
+  clear: both;
+  height: 0;
+}
```

Comment:
one more thing... this 'clearfix' is a commonly used practice across web development for using a class name to handle situations where they are using float properties for css positioning. It's kind of like a global rule too, so maybe if we are going to keep it you could move it into `index.css` or `App.css`

Reply:
removed it-- we actually didnt need it